pyproject.toml

```diff
@@ -14,7 +14,7 @@ authors = []
 dependencies = [
     "certifi>=2021.5.30",
     "importlib_metadata>=4.8.1",
-    "qiskit_serverless>=0.18.0, <0.19.0",
+    "qiskit_serverless @ git+https://github.com/Qiskit/qiskit-serverless.git#subdirectory=client",
```

Comment:
This change is just for the purpose of the PR to compare with our current development repository in serverless. It will need to be restore it before merge.